### ISSUE TITLE
List Linodes by tags returns error

### ISSUE THREAD

**Issue by dkam** [linode/linode-cli#151]
When viewing Linodes by tag, an error is returned.

```shell
$ linode-cli linodes list --tags mytag
Request failed: 400
┌errors────┬──────────────────────────────────────┐
│ field    │ reason                               │
├──────────┼──────────────────────────────────────┤
│ X-Filter │ Tags filter must be a string or dict │
└──────────┴──────────────────────────────────────┘
```

The error is returned for quoted strings also.

**Comment by Dorthu** [comment]
Thanks for reporting this - taking a look now

**Comment by Dorthu** [comment]
captured the request it was sending, and here's the X-Filter header (with `--tags demo`):

```
 'X-Filter': '{"tags": ["d", "e", "m", "o"]}'
```

That's .. not what I wanted.  I'll have a fix up in a bit

**Comment by Dorthu** [comment]
This is fixed in #152 - thanks again for reporting it!
